### PR TITLE
Add netsh command to enumerate processes using eBPF

### LINKED ISSUE #555: Ability to query what process(es) hold open fds/handles on a given program/map/link
The netsh helper has had the following TODO in it since the beginning:

> // TODO: see if the program is still loaded, in which case some other process holds
> // a reference. Get the PID of that process and display it.

However, this needs an API to query that information, which is required by bpftool (and netsh).

### PR DESCRIPTION
**Pull request by dthaler**
Fixes #555

## Description

Adds "netsh ebpf show processes" command to enumerate processes using eBPF.
This can be used to determine what needs to be stopped so that one can reload the ebpf driver.

## Testing

Added unit test.
Manual testing.

## Documentation

Netsh help output is updated.

### CODE REVIEW

File: libs/ebpfnetsh/processes.cpp [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        goto Done;
+    }
+    memcpy(&object_type_info, buffer, sizeof(object_type_info));
+
+    // eBPF handles are always of type "File".
+    if (wcscmp(object_type_info.TypeName.Buffer, L"File") != 0) {
+        goto Done;
+    }
+
+    // Set a 200ms timeout for querying object name after which we just give up.
```
Comment (Alan-Jowett) [inline libs/ebpfnetsh/processes.cpp:92]: File an issue to track this improvement.

**Review by Alan-Jowett — DISMISSED** [review]
Look good, some suggestions.

File: libs/ebpfnetsh/processes.cpp (hunk 1 above)
Comment (dthaler) [reply libs/ebpfnetsh/processes.cpp:92]: Where?  (btw, resource manager does the same thing here)

Comment (Alan-Jowett) [reply libs/ebpfnetsh/processes.cpp:96]: I was proposing filing an GH issue to track using a thread pool work item instead of spawning a thread each time.

**Review by Alan-Jowett — APPROVED** [review]

**Review by gtrevi — DISMISSED** [review]

**Review by Alan-Jowett — APPROVED** [review]

**Review by gtrevi — APPROVED** [review]

**Pull request merged**